tensorflow/python/eager/function.py

```diff
@@ -3241,14 +3255,16 @@ def _define_function_with_shape_relaxation(self, args, kwargs):
           (args, kwargs), relaxed_arg_specs, expand_composites=False)
       (args, kwargs) = nest.pack_sequence_as(
           (relaxed_arg_specs, relaxed_kwarg_specs),
-          nest.flatten((args, kwargs), expand_composites=True),
+          flat_args + flat_kwargs,
           expand_composites=True)
 
     graph_function = self._create_graph_function(
         args, kwargs, override_flat_arg_shapes=relaxed_arg_shapes)
     self._function_cache.arg_relaxed[rank_only_cache_key] = graph_function
 
-    return graph_function, args, kwargs
+    return (graph_function,
+            nest.flatten(args, expand_composites=True),
+            nest.flatten(kwargs, expand_composites=True))
```

Comment:
this is same as flat_args, right?

Reply:
It seems that args and kwargs can be modified/repacked in the `if any_composite_args` case (L3242/3256). Will check if they end up flattening to the same thing though